DrawerLayout.js

```diff
@@ -463,11 +494,15 @@ export default class DrawerLayout extends Component<PropType, StateType> {
 
     return (
       <PanGestureHandler
-        ref={this._panGestureHandler}
+        ref={this._setPanGestureRef}
         hitSlop={hitSlop}
         minOffsetX={gestureOrientation * minSwipeDistance}
+        maxDeltaY={15}
```

Comment:
this is also potentially controversial and perhaps should be a prop. the reason for adding it is that we need some criteria for drawer to fail at if we want to wait for drawer to fail before starting another gesture

Reply:
Hi, @brentvatne 
Could you please use https://kmagiera.github.io/react-native-gesture-handler/docs/handler-pan.html#maxdeltax instead of this old API?

Reply:
do you mean `failOffsetY`? if so, done. also changed `minOffsetX` to `activeOffsetX`

Reply:
yes, ofc